bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/ReminderView.java

```diff
@@ -122,7 +122,7 @@ public class ReminderView extends ViewPart implements IRefreshable, HeartListene
 	private Set<Integer> filterActionSet = new HashSet<Integer>();
 
 	private long cvHighestLastUpdate = 0l;
-
+	private boolean sortByDueDateSetting = ConfigServiceHolder.getUser(Preferences.USR_SORT_BY_DUE_DATE, false);
```

Comment:
Wenn da 0, 1 oder 2 in der Konfig Tabelle gespeichert wird, ist eine Konvertierung zu boolean nicht sauber.